src/openrpc/closed_captions.json

```diff
@@ -621,6 +621,50 @@
                     }
                 }
             ]
+        },
+        {
+            "name": "preferredLanguages",
+            "summary": "A prioritized list of ISO 639 1/2 codes for the preferred closed captions languages on this device.",
+            "params": [],
+            "tags": [
+              {
+                "name": "property"
+              },
+              {
+                "name": "capabilities",
+                "x-uses": [
+                  "xrn:firebolt:capability:accessibility:closedcaptions"
+                ]
+              }
+            ],
+            "result": {
+              "name": "languages",
+              "summary": "the preferred closed captions languages",
+              "schema": {
+                "type": "array",
+                "items": {
+                  "$ref": "https://meta.comcast.com/firebolt/localization#/definitions/ISO639Language"
```

Comment:
Suggest to call this: ISO639-2Language

Reply:
We've got an undocumented requirement that all schema names are safe variable names for JS & C.

Probably worth fixing in our code generation, but not on a Friday. Happy to look into it Monday if you still want the hyphen, but made it an underscore for now.

`ISO639_2Language`